tests/mock_tables/dbconnector.py

```diff
@@ -91,6 +91,7 @@ def __init__(self, *args, **kwargs):
         # to identify the file path to load the db json files.
         namespace = kwargs.pop('namespace')
         db_name = kwargs.pop('db_name')
+        self.decode_responses = kwargs.pop('decode_responses') == True
```

Comment:
Is the `== True` necessary?

Reply:
Yes. Caller could provide any type, so I want to normalize it into bool to make implementation simple.

---
In reply to: [510382170](https://github.com/Azure/sonic-snmpagent/pull/171#discussion_r510382170) [](ancestors = 510382170)